tests/tests_change_fs.yml

```diff
@@ -101,6 +101,22 @@
     - name: Verify role results
       include_tasks: verify-role-results.yml
 
+    - name: Remove the FS
+      include_role:
+        name: linux-system-roles.storage
+      vars:
+        storage_pools:
+          - name: foo
+            disks: "{{ unused_disks }}"
+            volumes:
+              - name: test1
+                size: "{{ volume_size }}"
+                fs_type: None
```

Comment:
Does this require `storage_safe_mode: false`?  Seems kinda dangerous

Reply:
Yes, it does. In code the behavior extends the functionality of `_reformat` method which handles the `safe_mode`.
Still, `fs_type: None` 'creation' on empty device will work even with `safe_mode` on since it is a no-op.